scripts/ckan_migration/migrate_harvest_sources.py

```diff
@@ -237,19 +273,46 @@ def upload_sources(sources):
     markers.
     """
     click.echo("Uploading sources...")
-    with ThreadPoolExecutor() as executor:
+    with ThreadPoolExecutor(max_workers=CONFIG.workers) as executor:
```

Comment:
what happens if `CONFIG.workers` is not set?

Reply:
It's set unconditionally in the main script with a default of `None` when the argument isn't given which gives the default behavior in `ThreadPoolExecutor`.

Reply:
I see it now!